R/susie_wrapper.R

```diff
@@ -166,7 +166,7 @@ susie_rss_qc <- function(z, R, ref_panel, bhat=NULL, shat=NULL, var_y=NULL, n =
 
   ## Perform initial SuSiE RSS analysis with discrepancy correction
   result <- susie_rss(z=z, R=R, bhat=bhat, shat=shat, var_y=var_y, n=n, L=max_L,  
-                     correct_zR_discrepancy=TRUE, track_fit = TRUE, ...)
+                     correct_zR_discrepancy=TRUE, track_fit = TRUE, max_iter = 100,...)
```

Comment:
@gaow The logic is: 
1. In susie_rss_qc() function, if we need imputation, it will run 2 susie_rss. first susie_rss with ZR_discrep = T is for detection of outlier.
2. The second susie_rss will run with imputed z scores.
3. However, if we need conditional analysis on imputed result. THe first round and second round susie_rss will both have max_iter = 1 as input (they all have `...` ). So that The detected outlier number will be influenced, to avoid that, I specified the max_iter = 100 (default) in the first round outlier.

Let me know if there are other decent solutions!